frontend/src/plugins/impl/DataTablePlugin.tsx

```diff
@@ -358,6 +364,9 @@ export const LoadingDataTableComponent = memo(
       }
     }, [props.pageSize, paginationState.pageSize]);
 
+    const containerRef = useRef<HTMLDivElement>(null);
+    const cellId = useFindCellId(containerRef);
```

Comment:
the plugin gets a prop called `hostElement`. i wonder if passing that down would be easier, and remove the `setTimeout`. (the `setTimeout` feels like something is wrong)